Content.Server/GameObjects/Components/AnchorableComponent.cs

```diff
@@ -14,6 +14,8 @@ public class AnchorableComponent : Component, IInteractUsing
     {
         public override string Name => "Anchorable";
 
+        int IInteractUsing.Priority { get => 1; }
```

Comment:
Can't you do `int IInteractUsing.Priority => 1;` here?

Reply:
I didn't know you could do that, I'll change those